frontend/app/views/jobs/_show_templates.html.erb

```diff
@@ -132,6 +132,15 @@
 
 <% end %>
 
+<%# This job type is hidden and will not appear in the job_types list so a template must be created individually %>
+<% define_template "top_container_linker_job", jsonmodel_definition(:top_container_linker_job) do |form, job| %>
+
+    <% begin %>
+      <%= render_aspace_partial :partial => "top_container_linker_job/show", :locals => {:job => job, :form => form} %>
+      <% rescue ActionView::MissingTemplate %>
```

Comment:
I am not sure what the intention is here. When I tested the linking job, this `rescue` did happen, since there is no such named template. I recommend taking the entire `begin` clause out if there is indeed no such template.